hrm_class.py

```diff
@@ -0,0 +1,231 @@
+import numpy
+import pandas
+
+class hrm_data:
```

Comment:
oddly enough in Python (PEP8), classes should be CapCased (so should be `HrmData` or `HRMData`: https://www.python.org/dev/peps/pep-0008/#class-names)